client/main.lua

```diff
@@ -355,6 +377,8 @@ RegisterNUICallback('select', function(data, cb)
     end
 
     if IsNuiFocused() then
+        activated = false
         state.setActive(false)
+        state.setNuiFocus(false)
```

Comment:
?????????

Still don't see what your stupid "activated" is for, and _how would somebody select if they don't have nui focus?_